pkg/rudder/client.go

```diff
@@ -17,21 +17,19 @@ limitations under the License.
 package rudder // import "k8s.io/helm/pkg/rudder"
 
 import (
-	"fmt"
-
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
 	"k8s.io/helm/pkg/proto/hapi/release"
 )
 
-// GrpcAddr is port number for accessing Rudder service
-var GrpcAddr = 10001
+// GrpcAddr is an address that should be used to access Rudder service
+const GrpcAddr = "127.0.0.1:10001"
```

Comment:
This is used in cmd/rudder/rudder.go (in main func). Does it work properly like that?